packages/overlay/src/overlay.ts

```diff
@@ -64,14 +64,14 @@ export class Overlay {
      * @param options.placement side on which to position the overlay
      * @returns an Overlay object which can be used to close the overlay
      */
-    public static open(
+    public static async open(
```

Comment:
This static method shares the same name as the `public async open` method in line 98.  The static method creates and opens an overlay, while the other opens an existing overlay. Should we rename one of them for clarity? 

Probably out of scope for this PR so feel free to ignore

Reply:
This makes sense to me. @cuberoot, I think you brought this to the project to begin with, do you have any thoughts here? If you think this makes sense too, I'll add an issue to track separately.

Something to keep in mind here is that this will be a breaking change...I don't think we have much usage of this API, but it would be good to look at doing this sort of change soon.